apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveDao.kt

```diff
@@ -183,6 +185,7 @@ class OppgaveDaoImpl(private val connection: () -> Connection) : OppgaveDao {
                         s.adressebeskyttelse is null OR 
                         (s.adressebeskyttelse is NOT NULL AND (s.adressebeskyttelse != ? AND s.adressebeskyttelse != ?))
                     )
+                    AND (? = true OR o.saksbehandler = ?)
```

Comment:
```suggestion
                    AND (? OR o.saksbehandler = ?)
```
burde være det samme

Reply:
Hvordan skal dette fungere...? 

`.setBoolean(8, minOppgavelisteIdentFilter == null)`

...skal sjekkes med:

`AND (? = true OR o.saksbehandler = ?)`

Det vil da stå `true = true`. 
Forslaget ditt @oyvindsh vil vel heller ikke gjøre noe som helst siden det vil bli: 
`AND (true OR o.saksbehandler = <ident>)`. Må da spesifisere _**hva**_ som skal være true.

Reply:
Done, nice catch